package.json

```diff
@@ -19,16 +19,7 @@
     "*.{json,css,md}": [
       "prettier --write"
     ]
-  },
-  "dependencies": {
-    "@types/bcryptjs": "^2.4.6",
-    "@types/jsonwebtoken": "^9.0.6",
-    "@types/passport-jwt": "^4.0.1",
-    "bcryptjs": "^2.4.3",
-    "jsonwebtoken": "^9.0.2",
-    "passport": "^0.7.0",
-    "passport-jwt": "^4.0.1",
-  }
```

Comment:
why are we removing these dependancies in this pr?

Reply:
we don't want deps in the root package.json

Reply:
we moved them to the backend package.json as they are only used in backend